File: javaaa/ClassObject.java
package javaaa;

class Product {
    String name;
    double price;

    Product(String name, double price) {
        this.name = name;
        this.price = price;
    }

    void displayInfo() {
        System.out.println(name + ": $" + price);
    }
}

public class ClassObject {
    public static void main(String[] args) {
        Product laptop = new Product("Laptop", 2000);
        laptop.displayInfo();
    }
}

File: darttt/ClassObject.dart
class Product {
    String name;
    double price;
  
    Product(this.name, this.price);

    void display() {
      print("Product Name: $name");
      print("Product Price: \$${price.toStringAsFixed(2)}");
    }
}

void main() {
  Product laptop = Product("Laptop", 1000.00);
  laptop.display();
}

